scene/resources/texture.cpp

```diff
@@ -959,6 +959,10 @@ bool AtlasTexture::has_filter_clip() const {
 	return filter_clip;
 }
 
+Ref<Image> AtlasTexture::get_data() const {
+	return atlas->get_data()->get_rect(region);
```

Comment:
This will crash if the atlas is invalid. You should check the validity of `atlas` before dereferencing it (see other methods in this class).

Reply:
done